packages/jobs/lib/runner/runner.ts

```diff
@@ -54,12 +55,17 @@ export async function getOrStartRunner(runnerId: string): Promise<Runner> {
             logger.error(err);
         }
     }
-    const isRender = process.env['IS_RENDER'] === 'true';
     let runner: Runner;
-    if (isRender) {
-        runner = await RenderRunner.getOrStart(runnerId);
-    } else {
-        runner = await LocalRunner.getOrStart(runnerId);
+    switch (envs.RUNNER_TYPE) {
```

Comment:
should we do toUpperCase to ensure it is consistent? Or is that handled already?

Reply:
already handled by z.enum. `envs.RUNNER_TYPE` is a union type `'LOCAL' | 'RENDER' | 'REMOTE'`